drltrace/CMakeLists.txt

```diff
@@ -109,3 +109,12 @@ endif ()
 install(TARGETS drltrace DESTINATION "${INSTALL_BIN}"
   PERMISSIONS ${owner_access} OWNER_EXECUTE GROUP_READ GROUP_EXECUTE
   WORLD_READ WORLD_EXECUTE)
+
+##################################################
+# drltrace test
+
+if (BUILD_TOOL_TESTS)
+  get_target_property(app_path drsyscall_app LOCATION${location_suffix})
+  get_target_property(drltrace_path drltrace LOCATION${location_suffix})
+  add_test(drltrace ${drltrace_path} -- ${app_path})
```

Comment:
You asked only to change commit message, there is no change for code. I have just added a single exit-only test for drltrace executable.